README.md

```diff
@@ -71,6 +71,14 @@ With the python 3.6+ required for running, the following should work on most sys
 
 If you open a different terminal window (or log-off/restart your computer), you'll need to run **step 2** of the above list again before running `zulip-term`, since that activates that virtual environment. You can read more about virtual environments in the [Python 3 library venv documentation](https://docs.python.org/3/library/venv.html).
 
+### Keeping your install up to date
+
+Stable releases are made available on PyPI and GitHub; to ensure you keep up to date with them we suggest checking those sites for updates.
+Stable releases are also announced on the #**announce** stream on the Zulip server https://chat.zulip.org, where you are welcome to make an account.
```

Comment:
Could note about email alerts from that?

Reply:
Hmm, the email alerts would be stream wide, so I am not sure completely either.

Reply:
We could ask users to set an alert word, say _ZT release_, and adhere to it while posting on the stream.

Reply:
I've updated this to mention the expected topic releases will be made in.